signac/syncutil.py

```diff
@@ -11,14 +11,14 @@
 
 logger = logging.getLogger('sync')
 logging.addLevelName(LEVEL_MORE, 'MORE')
-logging.MORE = LEVEL_MORE
+logging.MORE = LEVEL_MORE  # type: ignore
```

Comment:
Explanation for the ignore: Assigning `logging.MORE` (and `logger.more` below) is setting a property that doesn't exist in the original module / class. This kind of monkey-patch raises an error:
```python
signac/syncutil.py:14: error: Module has no attribute "MORE"
signac/syncutil.py:21: error: "Logger" has no attribute "more"
```